### ISSUE TITLE
Output is non deterministic (gridy avatars)

### ISSUE THREAD

**Issue by elierotenberg** [DiceBear/avatars#64]
Hello,

Although the docs mention that given the same seed, the same output should be generated, it appears that there is some implicit hidden state that makes output non deterministic, as demonstrated by the following example:
```js
const Avatars = require("@dicebear/avatars").default;
const sprites = require("@dicebear/avatars-gridy-sprites").default;

const avatars = new Avatars(sprites, {});

const seed = "hello world";

const svg1 = avatars.create(seed);
const svg2 = avatars.create(seed);

console.log(svg1 === svg2);
```
[See repl.it](https://repl.it/repls/ImaginativeMurkyTags)

The generated image *looks* the same but is not the byte-level identical, which breaks tests that test against a previous value.

I believe the issue comes for upstream [`gridy-avatars`](https://github.com/darosh/gridy-avatars/blob/master/src/avatars.js#L8).

Unless/until the upstream package is fixed, I came up with a dirty monkey-patch to "fix" it:
```ts
const fixGridyClipARegExp = /clipPath id="clip-a-([0-9]*)"/;
const fixGridy = (svg: string): string => {
  const match = svg.match(fixGridyClipARegExp);
  if (!match) {
    return svg;
  }
  const [, id] = match;
  const clipA = new RegExp(`clip-a-${id}`, "g");
  const clipB = new RegExp(`clip-b-${id}`, "g");
  return svg.replace(clipA, `clip-a-0`).replace(clipB, "clip-b-0");
};

const originalCreate = avatars.create.bind(avatars);
avatars.create = (...args) => {
  const svg = originalCreate(...args);
  return fixGridy(svg);
}
```
[See repl.it](https://repl.it/repls/AchingWhisperedPlanes)

Should I make a PR here?

**Comment by FlorianKoerner** [comment]
Many thanks for the detailed description of the problem. You are welcome to submit a Pull Request. Your solution looks good.

**Comment by elierotenberg** [comment]
This should do it!

**Comment by FlorianKoerner** [comment]
Released with version [4.1.0](https://github.com/DiceBear/avatars/releases/tag/v4.1.0)
